tests_integration/nativeapp/test_version.py

```diff
@@ -12,11 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from shlex import split
+from typing import Any, Optional
 
+from yaml import safe_dump, safe_load
+
+from snowflake.cli.api.project.util import (
+    is_valid_unquoted_identifier,
+    to_identifier,
+    unquote_identifier,
+)
 from tests.project.fixtures import *
 from tests_integration.test_utils import contains_row_with, row_from_snowflake_session
 
 
+def set_version_in_app_manifest(manifest_path: Path, version: Any, patch: Any = None):
```

Comment:
can we make these str and optional int?

Reply:
It's really `Optional[str|int|float]`, so I figured why bother...

Reply:
yeah for tests it's fine

Reply:
ok. I really meant it as a "provide anything you want and we'll write it to the file" because it's a test context.